src/javascript/ast-value.ts

```diff
@@ -230,3 +230,15 @@ export function getNamespacedIdentifier(
 }
 
 export const CANT_CONVERT = 'UNKNOWN';
+
+export function isLiteralExpression(expr: babel.Node): boolean {
+  if (babel.isLiteral(expr)) {
+    return true;
+  }
+
+  if (babel.isBinaryExpression(expr)) {
+    return isLiteralExpression(expr.left) && isLiteralExpression(expr.right);
+  }
+
+  return false;
```

Comment:
@merlinnot how about this? just a little more readable and figured it may as well become a "literal expression" check (simplifies the calling point too)

Reply:
in actual fact, i don't think we need this after all (@rictic?).

if we had an observer for some crazy insane reason, `'onProp' + [1] + 'Changed'`, its technically a literal in the end. maybe we should just care less and accept any `BinaryExpression` which begins with a literal?

Reply:
Hm... I’d love to sneak in template literals too, so one can do (I tend to do that actually):
```ts
static get observers() {
  return [
    `f(
      sooooo,
      many,
      props,
      here
    )`,
  ];
}
```

Reply:
+1 to adding untagged template literals. supporting them in expressionToValue should be pretty straightforward